### DISCUSSION TITLE
Not showing folders during "select existing" for M2M files

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#3153] [Archive]
Currently when we `Add Existing`, it's showing only files and not grouped by folders as are initially. 

I guess this is just a work in progress still?

**Comment by benhaynes** [comment]
Good point! The plan (eventually) is to add the folders into the Layout itself... so you end up with folders at the top of the files, just like Google Drive, etc. That's not technically on our road-map for now... but hopefully soon (and _definitely_ soon if it gets sponsored 😉 ).
